youtube-dl.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Paylists
+# 2019: https://www.youtube.com/playlist?list=PLhEVvej0BBEI3YMccR3_CLDiBSNvGFv5I
+# 2002: https://www.youtube.com/playlist?list=PLEVVO3zRKfv_XK06fOi41FQVATxgOHs6o
+
+year=2020
+playlist="https://www.youtube.com/playlist?list=PLEVVO3zRKfv_XK06fOi41FQVATxgOHs6o"
+source_directory="2020Lockout"
+dest_directory="2020"
+
+### Santatize Data Input ###
+echo -n "Before starting, copy list from Evernote to evernote-list.txt"
+pause
```

Comment:
`wait 2`

Reply:
Just wondering, @SeanLeftBelow, do you want the user to press `anykey` to continue, or just a short delay?

Reply:
anykey.

Reply:
This was a bunch of scripts that I merged all into one. I expect this script to break on no data.